CVLByExample/Summarization/GhostSummary/GhostSums/ghostSums.spec

```diff
@@ -0,0 +1,164 @@
+methods {
+    function ghostUpdater(address a, int n) external envfree;
+    function ghostUpdaterReverts(address a, int n) external envfree;
+}
+
+ghost mapping(address => int) ghostAddrToInt256;
+ghost mapping(uint8 => mapping(bytes4 => mapping(mathint => mapping(address => int8)))) ghostNestedMapping;
+persistent ghost mapping(address => int) persistentGhostAddrToInt256;
```

Comment:
persistent behave differently so lets have a copy of the spec with persistent mirror and show the difference